app/workers/codewars_recorder.rb

```diff
@@ -4,11 +4,18 @@ class CodewarsRecorder
 
   def perform(user_id, codewars_username)
     user = User.includes(:codewars).find(user_id)
-    codewars = HTTParty.get("http://www.codewars.com/api/v1/users/#{codewars_username}/code-challenges/completed")['data']
-    codewars.each do |codewar|
-      codewar["completedLanguages"].each do |language|
-        unless user.codewars.where(:slug=>codewar["slug"], :language=> language).present?
-          user.codewars.create(:slug => codewar["slug"], :language => language)
+    codewars_url = "http://www.codewars.com/api/v1/users/#{codewars_username}" \
+                   "code-challenges/completed"
+    codewars = HTTParty.get(codewars_url)['data']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.